Formula/vkectl.rb

```diff
@@ -0,0 +1,25 @@
+class Vkectl < Formula
+  desc "Command-Line Interface for VKE(VolcanoEngine Kubernetes Engine)"
+  homepage "https://github.com/volcengine/vkectl"
+  url "https://github.com/volcengine/vkectl/archive/refs/tags/v0.1.0.tar.gz"
+  sha256 "899a3c913f78a1442ef6e8afceba34cb36e842464388609ffa228e3a1514d208"
+  license "Apache-2.0"
+
+  depends_on "go" => :build
+
+  def install
+    system "go", "build", *std_go_args(ldflags: "-s -w -X github.com/volcengine/vkectl/pkg/version.version=v#{version}"), "./main"
+  end
+
+  test do
+    version_out = shell_output("#{bin}/vkectl version")
+    assert_match version.to_s, version_out
+
+    resource_help_out = shell_output("#{bin}/vkectl resource -h")
```

Comment:
Can you propose a more substantial test than checking the version or calling for help? These tests rarely ever fail even for broken installations.

>We want tests that don’t require any user input and test the basic functionality of the application. For example `foo build-foo input.foo` is a good test and (despite their widespread use) `foo --version` and `foo --help` are bad tests. 

Ref. https://docs.brew.sh/Formula-Cookbook#add-a-test-to-the-formula

Reply:
PTAL